config/instance.php

```diff
@@ -110,7 +110,8 @@
 
 	'user_filters' => [
 		'max_user_blocks' => env('PF_MAX_USER_BLOCKS', 50),
-		'max_user_mutes' => env('PF_MAX_USER_MUTES', 50)
+		'max_user_mutes' => env('PF_MAX_USER_MUTES', 50),
+		'max_domain_blocks' => env('PF_MAX_DOMAIN_BLOCKS', 50),
```

Comment:
This is hardcoded elsewhere to 400 I think?

Reply:
Nope, did you mean this?

https://github.com/pixelfed/pixelfed/blob/60e053c93671d0e790cde02aa1c51bc011cca17a/app/Http/Controllers/Api/V1/DomainBlockController.php#L70

The `400` there is the http status code to be returned

Reply:
Oooh! Ha, that's what I get for reviewing on mobile when sleepy